src/napari_matplotlib/base.py

```diff
@@ -126,16 +125,12 @@ def draw(self) -> None:
         """
 
     def apply_napari_colorscheme(self):
-        """
-        Apply napari-compatible colorscheme to the axes object.
-        """
+        """Apply napari-compatible colorscheme to the axes object."""
         if self.axes is None:
             return
-        # changing color of axes background to napari main window color
-        self.canvas.figure.patch.set_facecolor(NAPARI_WINDOW_COLOR)
-
-        # changing color of plot background to napari main window color
-        self.axes.set_facecolor(NAPARI_WINDOW_COLOR)
+        # changing color of axes background to transparent
+        self.canvas.figure.patch.set_facecolor("none")
+        self.axes.set_facecolor("none")
 
         # changing colors of all axes
```

Comment:
Bit of a napari noob here, but I'd guess that these lines 👇  don't work with [light themes](https://napari.org/stable/gallery/new_theme.html).